pkg/registry/apis/secret/contracts/inline.go

```diff
@@ -6,6 +6,9 @@ import (
 	common "github.com/grafana/grafana/pkg/apimachinery/apis/common/v0alpha1"
 )
 
+// InlineSecureValueSupport allows resources to manage secrets inline
+//
+//go:generate mockery --name InlineSecureValueSupport --structname MockInlineSecureValueSupport --inpackage --filename inline_mock.go --with-expecter
```

Comment:
You can remove this and the generated mock, since I put that in `
pkg/registry/apis/secret/inline.go` 👍